lib/roo_on_rails/sidekiq/queue_latency.rb

```diff
@@ -17,7 +17,17 @@ def normalised_latency
         metric.round(3)
       end
 
+      private
+
       def permitted_latency
+        entries = ENV.fetch('SIDEKIQ_PERMITTED_LATENCY_VALUES', '').split(',')
+        entry = entries.find { |e| e.start_with?(queue.name) }
```

Comment:
Would probably be better to do this once and memoise as a `Hash` rather than doing the parsing each time.